pkg/controller/restore_controller.go

```diff
@@ -113,22 +112,22 @@ func NewRestoreController(
 
 ) Interface {
 	c := &restoreController{
-		namespace:           namespace,
-		restoreClient:       restoreClient,
-		backupClient:        backupClient,
-		restorer:            restorer,
-		objectStoreConfig:   objectStoreConfig,
-		bucket:              bucket,
-		pvProviderExists:    pvProviderExists,
-		backupLister:        backupInformer.Lister(),
-		backupListerSynced:  backupInformer.Informer().HasSynced,
-		restoreLister:       restoreInformer.Lister(),
-		restoreListerSynced: restoreInformer.Informer().HasSynced,
-		queue:               workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "restore"),
-		logger:              logger,
-		logLevel:            logLevel,
-		pluginRegistry:      pluginRegistry,
-		metrics:             metrics,
+		namespace:                  namespace,
+		restoreClient:              restoreClient,
+		backupClient:               backupClient,
+		restorer:                   restorer,
+		backupLocationLister:       backupLocationInformer.Lister(),
```

Comment:
easier if we keep these in the same order as the fields on the struct, so pls move `backupLocationLister` and `backupLocationListerSynced` below `restoreListerSynced`